tests/test.rs

```diff
@@ -18,7 +29,7 @@ fn create_socketpair() -> (Socket, Socket) {
     receiver.set_rcvtimeo(1000).unwrap();
 
     receiver.bind("tcp://*:*").unwrap();
```

Comment:
I'd rather avoid having platform-specificality in the test suite, if
it can be avoided. However, I'm not what the "proper" solution looks
like, or even if there is one. I'll have to look into this more
closely.